libs/ui/search/src/lib/record-thumbnail/record-thumbnail.component.html

```diff
@@ -15,9 +15,10 @@
     />
   </svg>
```

Comment:
This SVG element will never show up with the logic in this PR; this SVG is IMO the placeholder that's supposed to be made customizable

Reply:
See comment above, it's still displaying during image fetching.